qt/applications/workbench/workbench/plotting/backend_workbench.py

```diff
@@ -24,7 +24,9 @@
 # -----------------------------------------------------------------------------
 FigureCanvas = MantidFigureCanvas
 draw_if_interactive = QAppThreadCall(draw_if_interactive_impl)
+draw_if_interactive.__signature__ = signature(draw_if_interactive_impl)
```

Comment:
Just out of interest - what issues did the incorrect signature have? Is it problems with the completion?

Reply:
This seems to stop workbench launching with:  

```
File "/usr/local/Caskroom/miniconda/base/envs/mantid/lib/python3.8/inspect.py", line 1820, in _signature_bound_method
    raise ValueError('invalid method signature')
```
This comes about as matplotlib now explicitly tried to copy over the signature of the backend methods:

Before: https://github.com/matplotlib/matplotlib/blob/782f6ef56acca21a4e1057ef3e9aecf5e22e88a2/lib/matplotlib/pyplot.py#L242
Now: https://github.com/matplotlib/matplotlib/blob/422e5374f04423bac416ec6c9b8afa2d619f5285/lib/matplotlib/pyplot.py#L295 

Notice the addition of an `inspect.signature` method. I guess the `QAppThreadCall` masks the true signature?